src/lib/data/templates.ts

```diff
@@ -0,0 +1,7 @@
+export const COLLECTIONS = ['Best Sellers', 'Trending', 'Most Recent'];
+
+export const GENDERS = ['Male', 'Female', 'Genderless'];
+
+export const BRANDS = ['Spaarkd'];
```

Comment:
Is this necessary? Are our Templates branded? Even if they are, the "Brand" should come from the API, right?